aiogram/api/types/user.py

```diff
@@ -33,6 +41,80 @@ class User(TelegramObject):
 
     @property
     def full_name(self) -> str:
+        """
+        Get full name of user.
+        :return: str
+        """
         if self.last_name:
             return f"{self.first_name} {self.last_name}"
         return self.first_name
+
+    @property
+    def mention(self) -> str:
+        """
+        Get user's username to mention him.
+        Full name will be returned if user has no username.
+        :return: str
+        """
+        if self.username:
+            return "@" + self.username
+        return self.full_name
+
+    @property
+    def locale(self) -> Optional[babel.core.Locale]:
```

Comment:
I think this property can be removed in 3.0. Need to be discussed.

Reply:
Already 4 votes for removal and no argument for saving. It might be worth dropping this commit.